src/soundvectors.py

```diff
@@ -406,125 +506,108 @@ def validate(self, sound):
                     return sound_transcribed.name
         raise ValueError("Invalid sound encountered.")
 
-    def get_vec(self, sound, vectorize=True):
-        """
-        Retrieve the vector for a given sound.
-
-        @param sound: A sound represented by a feature string. If a
-            transcription system has been supplied, it can also be a 
-            sound in IPA notation.
-        @param vectorize: Return a binary vector if set to True.
-        @returns: vector (dictionary of feature values or tuple of values)
-        
-        Examples::
-            
-            >>> from soundvectors import SoundVectors
-            >>> sv = SoundVectors()
-            >>> vec = sv.get_vec("t", vectorize=False)
-            >>> print(vec["cont"])
-            - 1
-        """
+    def __getitem__(self, sound: typing.Union[str, Sound]) -> FeatureBundle:
```

Comment:
I've thrown in `__getitem__` to retrieve a `FeatureBundle` instance. `get_vec` still behaves as before.